src/crate/crash/repl.py

```diff
@@ -194,6 +195,9 @@ def get_completions(self, document, complete_event):
             return
         line = document.text
         word_before_cursor = document.get_word_before_cursor()
+        # Show completion only if at least 1 character has been typed
```

Comment:
autocompletion on 3rd character is already in master. https://github.com/crate/crash/commit/6a6070a14ce86a83cdddc8b7d60cd2cb225e8016  --> pls rebase.